Resources/Prototypes/explosion.yml

```diff
@@ -77,6 +78,7 @@
   fireColor: Blue
   texturePath: /Textures/Effects/fire_greyscale.rsi
   fireStates: 3
+  fireStacks: 0
```

Comment:
This isn't necessary, the float being nullable and the nullcheck takes care of it for you.